From 7a2b4a58ded13d28d82fc203e9744d28b447d2cd Mon Sep 17 00:00:00 2001
From: Rex Zhu <Rex.Zhu@amd.com>
Date: Wed, 12 Apr 2017 13:40:26 +0800
Subject: [PATCH 1191/2094] Revert "drm/amd/powerplay: set sensitivity of fan
 reaction to temepature changes."

This reverts commit 8d42471937505338eeac4e98894f9c4c770aa3c5.

revert this patch as typo in the patch title "temepature"

Change-Id: I66dd376516cee68c2f1cf72a90133d97405e01a8
---
 drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c
index 4ada673..f4d77b6 100644
--- a/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c
+++ b/drivers/gpu/drm/amd/powerplay/hwmgr/vega10_thermal.c
@@ -553,8 +553,6 @@ int tf_vega10_thermal_setup_fan_table(struct pp_hwmgr *hwmgr,
 	if (!data->smu_features[GNLD_FAN_CONTROL].supported)
 		return 0;
 
-	table->FuzzyFan_PwmSetDelta = (uint16_t)hwmgr->thermal_controller.
-			advanceFanControlParameters.usFanOutputSensitivity;
 	table->FanMaximumRpm = (uint16_t)hwmgr->thermal_controller.
 			advanceFanControlParameters.usMaxFanRPM;
 	table->FanThrottlingRpm = hwmgr->thermal_controller.
-- 
2.7.4

